runtime/bindings/python/tests/test_inference_engine/test_infer_request.py

```diff
@@ -163,6 +163,6 @@ def test_infer_mixed_keys(device):
     tensor2 = Tensor(data2)
 
     request = exec_net.create_infer_request()
-    with pytest.raises(TypeError) as e:
+    with pytest.raises(KeyError) as e:
```

Comment:
But ```typeError``` comes from https://github.com/akuporos/openvino/pull/58/files?authenticity_token=GzJkbpc%2F1U9xlNck0DHfaK96NSHx4I0HII7T6nZbn3Oict29LuBQi9LT4%2BoVpcXORu2HbOeXFjRy9TLvUzZqFA%3D%3D&file-filters%5B%5D=.cpp&file-filters%5B%5D=.py#diff-8d265038075a6c6d31396b3a6fe6222dddd4237ed6a74f0f1938e9a776bb495dR297

Reply:
Ok, from ```normalize_inputs``` also TypeError is thrown